x-pack/plugins/data_visualizer/common/constants.ts

```diff
@@ -27,7 +29,26 @@ export const JOB_FIELD_TYPES = {
   KEYWORD: 'keyword',
   NUMBER: 'number',
   TEXT: 'text',
+  HISTOGRAM: 'histogram',
   UNKNOWN: 'unknown',
 } as const;
 
+export const JOB_FIELD_TYPES_OPTIONS = {
+  [JOB_FIELD_TYPES.BOOLEAN]: { name: 'Boolean', icon: 'tokenBoolean' },
+  [JOB_FIELD_TYPES.DATE]: { name: 'Date', icon: 'tokenDate' },
+  [JOB_FIELD_TYPES.GEO_POINT]: { name: 'Geo point', icon: 'tokenGeo' },
+  [JOB_FIELD_TYPES.GEO_SHAPE]: { name: 'Geo shape', icon: 'tokenGeo' },
+  [JOB_FIELD_TYPES.IP]: { name: 'IP address', icon: 'tokenIP' },
+  [JOB_FIELD_TYPES.KEYWORD]: { name: 'Keyword', icon: 'tokenKeyword' },
+  [JOB_FIELD_TYPES.NUMBER]: { name: 'Number', icon: 'tokenNumber' },
+  [JOB_FIELD_TYPES.TEXT]: { name: 'Text', icon: 'tokenString' },
+  [JOB_FIELD_TYPES.HISTOGRAM]: { name: 'Histogram', icon: 'tokenNumber' },
```

Comment:
Can you swtch this to the `tokenHistogram` icon from the EUI token icon set.

Reply:
The `histogram` type should also be added to the field type dropdown:

![image](https://user-images.githubusercontent.com/7405507/124776380-4f34fc80-df37-11eb-8275-da79ebdca677.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/104553/commits/48a55e87ca3532885807a393c00d23224b1adef1